testutils/clustermanager/gke.go

```diff
@@ -45,8 +46,9 @@ var (
 	protectedProjects       = []string{"knative-tests"}
 	protectedClusters       = []string{"knative-prow"}
 	// These are arbitrary numbers determined based on past experience
-	creationTimeout = 20 * time.Minute
-	deletionTimeout = 10 * time.Minute
+	creationTimeout    = 20 * time.Minute
+	deletionTimeout    = 10 * time.Minute
+	autoscalingTimeout = time.Minute
```

Comment:
Just to be explicit.

```suggestion
	autoscalingTimeout = 1 * time.Minute
```